spec/unit/policyfile_evaluation_spec.rb

```diff
@@ -533,6 +533,19 @@
 
     end
 
+    describe "the metadata sugar keyword" do
+      let(:policyfile_rb) do
+        <<-EOH
+          run_list "foo"
+          metadata
+        EOH
+      end
+      it "reads the metadata from the current cookbook" do
+        puts Dir.pwd
```

Comment:
Debugging?

Reply:
yeah had an unpushed commit of half-written specs that got pushed after i rebased...  still need to finish this up